addons/datadog/values.yaml

```diff
@@ -796,7 +796,7 @@ datadog:
   # Autodiscovery, as a space-sepatered list
 
   ## ref: https://docs.datadoghq.com/agent/guide/autodiscovery-management/?tab=containerizedagent#exclude-containers
-  containerExclude: 'kube_namespace:cert-manager kube_namespace:ingress-nginx kube_namespace:kube-system kube_namespace:monitoring kube_namespace:porter-agent-system'
+  containerExclude: 'kube_namespace:cert-manager kube_namespace:ingress-nginx kube_namespace:kube-system kube_namespace:monitoring kube_namespace:porter-agent-system kube_namespace: telemetry'
```

Comment:
So the format here is `kube_namespace:<NAMESPACE>` - no spaces between them, think DD acts up otherwise